frontend/dockerfile/linter/ruleset.go

```diff
@@ -132,4 +132,12 @@ var (
 			return fmt.Sprintf("Setting platform to predefined %s in FROM is redundant as this is the default behavior", platformVar)
 		},
 	}
+	RuleInvalidDefaultArgInFrom = LinterRule[func(string) string]{
+		Name:        "InvalidDefaultArgInFrom",
+		Description: "Using ARG with default value results in an empty or invalid base image name",
+		URL:         "https://docs.docker.com/go/dockerfile/rule/invalid-default-arg-in-from/",
+		Format: func(baseName string) string {
```

Comment:
nit: it would be a bit more user-friendly if we would detect here if the base image is invalid or empty and show a message that is precise to the current condition. In case of "invalid image" we could also show what that invalid image name is.